src/components/packages/Package-writing.svelte

```diff
@@ -0,0 +1,37 @@
+<style>
+    .text{
+        display: flex;
+        flex-direction: column;
+        width: 60%;
+        margin: 0 auto;
+        padding-left: 5px;
+        padding-right: 5px;
+    }
+
+    .text-body{
+        display:inline-block;
+        text-align: left;
+    }
+
+    h1{
+        font-family: 'Vollkorn SC', serif;
+        font-size: 8vh;
+        color: rgb(68, 91, 71)
```

Comment:
Make sure to use the color variables instead of hard coded color values. This way ff the color changes at some point we can change it once at top level. Should look something like this:
`color: var(--sg-green);`